cohorts/views.py

```diff
@@ -1555,52 +1566,62 @@ def write(self, value):
 
 def streaming_csv_view(request, cohort_id=0):
```

Comment:
IMO maybe make a ticket for evaluating if anything which probably needs login_required doesn't have it. (Not something I want to get into for this sprint since it's app-wide.)